accord-core/src/main/java/accord/utils/btree/BTreeRemoval.java

```diff
@@ -68,6 +73,51 @@ public static <V> Object[] remove(final Object[] btree, final Comparator<? super
         return result;
     }
 
+    /**
+     * Remove |elem| from |btree|. If it's not present then return |btree| itself.
+     */
+    public static <F, V> Object[] remove(final Object[] btree, final AsymmetricComparator<? super F, ? super V> comparator, final F elem)
```

Comment:
This is is diverging from the C* version now, the unit tests aren't in Accord and this doesn't have a unit test.

This looks like a cut and paste of the other comparator version so not very serious, but still.

Reply:
I don't like that we duplicate it at all, really. But not sure when to find the time to create a utilities subproject we can share this through